src/main/java/brave/opentracing/BraveSpanBuilder.java

```diff
@@ -167,4 +173,78 @@
 
     return result;
   }
+
+  static class TagSetterHack<T> implements Span {
```

Comment:
hack for https://github.com/opentracing/opentracing-java/issues/340 cc @tylerbenson